pages/_app.tsx

```diff
@@ -151,13 +165,15 @@ function MyApp({ Component, pageProps }: ComponentWithPageLayout) {
         <SessionContextProvider supabaseClient={supabaseClient} initialSession={pageProps.initialSession}>
           <PostHogProvider client={posthog}>
             <TipProvider>
-              {Component.PageLayout ? (
-                <Component.PageLayout>
+              <FaroErrorBoundary>
```

Comment:
I'm not sure this is the right place for the error boundary ... their React docs make it seem like this should wrap a rendered `<App />` and I'm not seeing errors flow through when testing.